### PR TITLE
Parameterize XSLT files and branches

### LINKED ISSUE #26: set up way to pass in the XSLT filename as a parameter to transform task
Namely, the process is like this:
1. There is an Airflow Variable (`???_OAI_CONFIG` key perhaps?) pointing to the transform filename to be used from https://github.com/tulibraries/aggregator_mdx.git (like `villanova.xsl`)
2. The BashOperator XSLT_TRANSFORM task then knows to use that file from aggegator_mdx/transform/{FILENAME} in the transform script

### PR DESCRIPTION
**Pull request by bibliotechy**
Closes #26 and #27

### CODE REVIEW

File: funcake_villanova_dag.py [hunk 1]
```diff
@@ -21,12 +21,20 @@
...
+#   "schematron_filename": "validations/test_validation", <--- OPTIONAL
 # }
-MDX_PREFIX = VILLANOVA_OAI_CONFIG.get("md_prefix")
+MDX_PREFIX   = VILLANOVA_OAI_CONFIG.get("md_prefix")
 INCLUDE_SETS = VILLANOVA_OAI_CONFIG.get("included_sets")
 OAI_ENDPOINT = VILLANOVA_OAI_CONFIG.get("endpoint")
+XSL_BRANCH   = VILLANOVA_OAI_CONFIG.get("xsl_branch", "master")
+XSL_FILENAME = VILLANOVA_OAI_CONFIG.get("xsl_filename", "transforms/villanova.xsl")
+XSL_REPO     = VILLANOVA_OAI_CONFIG.get("xsl_repo", "tulibraries/aggregator_mdx")
+SCHEMATRON_FILENAME = VILLANOVA_OAI_CONFIG.get("schematron_filename", "validations/padigital_reqd_fields.sch")
```
Comment (cmharlow) [inline funcake_villanova_dag.py]: Wondering if we should have a separate `VILLANOVA_XSLT_CONFIG` for two reasons: 
1. will work with CSV harvests
2. we can attach different filtering schematrons to both the HARVEST and the XSLT tasks.

Comment (bibliotechy) [reply funcake_villanova_dag.py]: @cmharlow Yeah, I was wondering about that. I'll make that change

Comment (cmharlow) [reply funcake_villanova_dag.py]: thanks @bibliotechy

**Comment by bibliotechy** [comment]
@cmharlow Changes addressed.

**Comment by cmharlow** [comment]
Probably need to change the XSLT Schematron variable, but can address that in the next PR

**Pull request merged**